app/models/base_organisation.rb

```diff
@@ -44,4 +44,13 @@ def gmaps4rails_marker_attrs
   def has_been_updated_recently?
     updated_at >= 1.year.ago
   end
+  
+  def slug_candidates
+    # It checks if self.name is nil, because otherwise it breaks -
```

Comment:
For me it wasn't clear. The problem appeared when I run the specs. So I put
it there for spare somebody in the future considerations about - "is this
self.name necessary?"

I think it's useful.

2016-05-09 11:48 GMT+02:00 Sam Joseph <notifications@github.com>:

> In app/models/base_organisation.rb
> <https://github.com/AgileVentures/LocalSupport/pull/292#discussion_r62473930>
> :
>
> > @@ -44,4 +44,13 @@ def gmaps4rails_marker_attrs
> >    def has_been_updated_recently?
> >      updated_at >= 1.year.ago
> >    end
> > +
> > +  def slug_candidates
> > +    # It checks if self.name is nil, because otherwise it breaks -
>
> I wonder if we need this comment?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/AgileVentures/LocalSupport/pull/292/files/bf7b03ed1d65dedf261c812acd715258d6344307#r62473930>
>